### PR TITLE
Hotfix for Stream Editor name/description inputs

### PR DESCRIPTION
**Pull request by fonty1**
Not sure why this is broken ATM the current code LGTM, but I'm obviously missing something!

This is a quick hotfix around the issue so users can edit the fields.

### CODE REVIEW

**Comment by fonty1** [comment]
> There's a more serious bug in there. @fonty1 I'mma try to fix it and push directly to your branch. Hope that's fine.

Go for it sir, looking forward to seeing your magic!

File: app/src/userpages/modules/userPageStreams/reducer.js [hunk 1]
```diff
@@ -271,12 +269,13 @@ export default function (state: UserPageStreamsState = initialState, action: Str
...
+            return {
                 ...state,
+                editedStream: {
+                    ...state.editedStream,
+                    [action.field]: action.data,
+                },
             }
-            const fullPath = `editedStream.${action.field}`
-            set(newState, fullPath, action.data)
-            return newState
```
Comment (mondoreale) [inline app/src/userpages/modules/userPageStreams/reducer.js]: Here! `newState.editedStream` was ALWAYS referencing the same object in memory. This is the reason why `useSelector` ignored updates to both name and description.

`useSelector` takes a second argument – `equalityFn`, `(a, b) => a === b` by default. `A === A` is always true. And using [`shallowEqual`](https://github.com/reduxjs/react-redux/blob/master/src/utils/shallowEqual.js) won't work either for obvious reasons (it's `A === A` but proves that one can write a loop).

The only answer is to.. update the store properly.

A cute little buggy bug. 🐞

**Comment by mondoreale** [comment]
@fonty1 please merge if you find my explanation reasonable and complete. Cheers!

File: app/src/userpages/modules/userPageStreams/reducer.js (hunk 1 above)
Comment (fonty1) [reply app/src/userpages/modules/userPageStreams/reducer.js]: Ahh interesting! Nice one.. I need to learn up on how this useSelector works with state updates..

Although seems to affect the configure fields section below which uses the same `UPDATE_EDIT_STREAM_FIELD` - if you 'Add field' it doesn't update the config fields state properly:

Dev branch:
Screenshot 2019-12-17 at 20 51 50

This branch:
Screenshot 2019-12-17 at 20 49 25

The `development` branch has it's own bugs of course I noticed as well after deploying to production :P Seems as though the hooks refactoring has had some collateral damage!
https://streamr.atlassian.net/browse/PLATFORM-1183

**Comment by mondoreale** [comment]
Looking into the other bug now – [`PLATFORM-1183`](https://streamr.atlassian.net/browse/PLATFORM-1183).

**Comment by mondoreale** [comment]
Oh, that's bad. The fix I made here doesn't work with the trickery made in ConfigureView. 😱 On it!

**Pull request merged**